CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 * [ENHANCEMENT] Query-frontend: track `cortex_frontend_query_response_codec_duration_seconds` and `cortex_frontend_query_response_codec_payload_bytes` metrics to measure the time taken and bytes read / written while encoding and decoding query result payloads. #4110
 * [ENHANCEMENT] Alertmanager: expose additional upstream metrics `cortex_alertmanager_dispatcher_aggregation_groups`, `cortex_alertmanager_dispatcher_alert_processing_duration_seconds`. #4151
 * [ENHANCEMENT] Store-gateway: use more efficient chunks fetching and caching. This should reduce CPU, memory utilization, and receive bandwidth of a store-gateway. #4163
+* [ENHANCEMENT] Query-frontend: Wait for in-flight queries to finish before shutting down. Only applicable when the query-frontend is used with the query scheduler. #4073
```

Comment:
We can remove "Only applicable when the query-frontend is used with the query scheduler." part, as this now applies for all frontend modes.

Reply:
Done, thanks!